docs/user-guide/src/bmo/firmware_components.md

```diff
@@ -0,0 +1,159 @@
+# Firmware Components
+
+Metal3 supports modifying firmware components of the bare metal hosts.
+This feature can be used to update the firmware for BIOS or BMC.
```

Comment:
"update the system firmware (e.g. BIOS) or BMC firmware" (BIOS is not a thing for a lot of hardware any more or at all)